Enterprise-CMCS/macpro-mdct-qmr
# Add Unique Identifiers to Aria Labels - (508 Remediation)

### Description
Add descriptive aria labels to action menus on and delete button ( uploads ).


### Related ticket(s)
[MDCT-](https://qmacbis.atlassian.net/browse/MDCT-1950)1950

---
### How to test
1. Go to core set and measure action menus and inspect the aria labels for unique identifiers.
2. Upload a pdf and check the delete button for aria label 

#### Core Set Ex.
<img width="1265" alt="Screenshot 2023-05-31 at 2 10 06 PM" src="https://github.com/Enterprise-CMCS/macpro-mdct-qmr/assets/14514294/75a5f8b8-6d64-4a56-9e6e-f690fda8ac39">

<img width="1387" alt="Screenshot 2023-05-31 at 2 11 22 PM" src="https://github.com/Enterprise-CMCS/macpro-mdct-qmr/assets/14514294/e71aa87d-3d53-4959-b46f-2c1abdb41358">

#### Measure Ex.
<img width="1222" alt="Screenshot 2023-05-31 at 2 11 48 PM" src="https://github.com/Enterprise-CMCS/macpro-mdct-qmr/assets/14514294/7a42cc2a-1fdd-44e1-81c2-69388d411db8">

<img width="1264" alt="Screenshot 2023-05-31 at 2 12 11 PM" src="https://github.com/Enterprise-CMCS/macpro-mdct-qmr/assets/14514294/09114d48-02ee-4fe7-aca1-c3d24b58c79c">

#### Delete Attachment Ex.
<img width="1325" alt="Screenshot 2023-05-31 at 3 43 36 PM" src="https://github.com/Enterprise-CMCS/macpro-mdct-qmr/assets/14514294/3ef50d85-0617-46c1-a71b-488c66ce4ffa">


---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
